source/content/guides/drush/02-drush-versions.md

```diff
@@ -12,54 +12,77 @@ cms: [drupal]
+**For most teams running a modern version of Drupal, the site-local version of Drush is only one to pay attention to.**
+The [`composer.json` file used by newly created Drupal 9 and 10 sites](https://github.com/pantheon-upstreams/drupal-composer-managed/blob/main/composer.json) is set to allow Drush 11 or 12 as the site-local version.
 
-## Verify Current Drush Version
+#### Changing the site-local Drush version
 
-You can use [Terminus](/terminus/) to verify the current version of Drush running on your Pantheon site:
+To change the site-local version of Drush run a composer command to update the version of Drush in the `composer.json` file. For instance, to set the site-local version of Drush to 11, run the following command:
 
-```bash{promptUser: user}
-terminus drush <site>.<env> -- status | grep "Drush version"
+```bash
+composer require drush/drush:^11
 ```
 
-## Compatibility and Requirements
-
-<Partial file="drush-compatibility.md" />
+### Drupal 7
 
-## Configure Drush Version
+For older versions of Drupal (7 and earlier), the global installation of Drush is the only one available.
 
-1. Refer to [Available Drush Versions](#available-drush-versions) and [Drush requirements](#compatibility-and-requirements) before you modify a site's Drush version. Remember that not all versions of Drush are compatible with all versions of Drupal.
+#### Changing the Global Drush version on Pantheon
 
-1. Change your site's Drush version in the [`pantheon.yml` file](/pantheon-yml/):
+To change the global Drush version (which will still invoke the site-local version of Drush when available), update the `drush_version` key in the `pantheon.yml` file. For instance, to set the global version of Drush to 11, add the following to the `pantheon.yml` file:
 
-    ```yaml:title=pantheon.yml
-    api_version: 1
+```yaml:title=pantheon.yml
+api_version: 1
 
-    drush_version: 8
-    ```
+drush_version: 8
+```
 
-Now your site’s Drush version is managed via `pantheon.yml`. This allows Drush to be version controlled and deployed along with the rest of your code.
+<Alert title="Note: Limited Drush Versions in Pantheon.yml" type="info">
 
+Even though Drush 11 and above are available for site-local installations, they are not available global version options in `pantheon.yml`. This restriction is present to reduce unnecessary or possibily counterproductive settings in `pantheon.yml`
 
-<Alert title="Note"  type="info" >
+If you do attempt to set a Drush version that is not supported, you will see an error message like the following:
 
-Create the `pantheon.yml` file if it does not already exist. If a `pantheon.upstream.yml` file exists, do not edit it. It is used by the upstream updates repository and will cause a [merge conflict if modified](/core-updates#error-updating-conflict-modifydelete-pantheonupstreamyml-deleted-in-head-and-modified-in-upstreammaster-version-upstreammaster-of-pantheonupstreamyml-left-in-tree).
+```bash
+remote:
```

Comment:
@rachelwhitton I thought it was appropriate to bring in this error message to improve searchablility.